provision/etc/dnsmasq-template.conf

```diff
@@ -0,0 +1,21 @@
+listen-address=%{IPADDR}
+
+log-queries
+log-facility=/var/log/dnsmasq.log
+
+enable-tftp
+tftp-root=%{TFTPROOT}
+
+dhcp-match=set:ipxe,175
+pxe-service=tag:!ipxe,X86-64_EFI,"EFI x86-64",/warewulf/ipxe/bin-x86_64-efi/snp.efi
+pxe-service=tag:!ipxe,BC_EFI,"EFI BC",/warewulf/ipxe/bin-x86_64-efi/snp.efi
+pxe-service=tag:!ipxe,IA32_EFI,"EFI IA32",/warewulf/ipxe/bin-i386-efi/snp.efi
+pxe-service=tag:!ipxe,x86PC,"x86 PC",/warewulf/ipxe/bin-i386-pcbios/undionly.kpxe
```

Comment:
Is there a aarch64 rule we can use here?

Reply:
According to dnsmasq's manpage: "The known types are x86PC, PC98, IA64_EFI, Alpha, Arc_x86, Intel_Lean_Client, IA32_EFI, X86-64_EFI, Xscale_EFI, BC_EFI, ARM32_EFI and ARM64_EFI; an integer may be used for other types."

Guess that would be ARM64_EFI then...

Reply:
Agreed ARM64_EFI seems correct.

Reply:
Curious the difference between dhcp-boot and pxe-service options. Seemingly from the man page we should be use dhcp-boot instead of pxe-server.

Reply:
Yes. It's a bit more complicated because the system type has to extracted manually, but it solves the problem with the .0 suffix.

Reply:
Awesome.